### DISCUSSION TITLE
regular expression search

### DISCUSSION THREAD

**Discussion by alfabetta** [sumatrapdfreader/sumatrapdf#4330] [Ideas]
My idea is regular expression search in sumatrapdf.

**Comment by GitHubRulesOK** [comment]
RE does not work well with non Latin languages
how do you define starts with `a` and ends with `ش` when both  of those are exactly the same letter so will match `a to a` 
nor with PDF space less construction, (it is not necessary in pdf to have word spaces) when searching for a small inline sequence such as found in single lines of text
Attempting RE beyond a single line is usually a fail. as there are no such things as sequential lines of text.

Nothing about PDFs is regular for expressions to be working it can only work after text has been extracted and that rarely works 

Best way to use RE is copy the page via the clipboard into an editor with RE ability, not try to do it in a reader..
